operations/configuration_management/iSCSI_SBPS_Node_Personalization.md

```diff
@@ -0,0 +1,359 @@
+
+ncn-m001:/home/csm-config-management #
+```
+
+### **View CFS session / check status**
+
+See [View CFS Sessions](CFS_Sessions.md) for more info.
+
+**E.g.**
+
+```
+ncn-m001:/home/csm-config-management # cray cfs v3 sessions describe  iscsi-sbps-targets-config
+debug_on_failure = false
+logs = "ara.cmn.surtur.hpc.amslabs.hpecorp.net/?label=iscsi-sbps-targets-config"
+name = "iscsi-sbps-targets-config"
+
+[ansible]
+config = "cfs-default-ansible-cfg"
+limit = ""
+passthrough = ""
+verbosity = 0
+
+[configuration]
+limit = ""
+name = "iscsi-sbps-targets-config"
+
+[status]
+artifacts = []
+
+[tags]
+
+[target]
+definition = "dynamic"
+groups = []
+image_map = []
+
+[status.session]
+completion_time = "2024-08-31T21:09:19"
+job = "cfs-b840455c-e919-4656-b64a-44d433f082dc"
+start_time = "2024-08-31T21:08:47"
+status = "complete"
+succeeded = "True"
+
+ncn-m001:/home/csm-config-management #
+```
+
+### **Track the Status of Ansible playbooks**
+
+Make sure that all the CFS play books have completed successfully.
+Look for the message "All playbooks completed successfully" in the cfs ansible container log.
+
+See [Troubleshooting](Track_the_Status_of_a_Session.md) for more info.
+
+**E.g.**
+
+```
+ncn-m001:/home/csm-config-management #  kubectl get pods --no-headers -o custom-columns=":metadata.name" -n services -l cfsession=iscsi-sbps-targets-config
+cfs-b840455c-e919-4656-b64a-44d433f082dc-5d4jp
+ncn-m001:/home/csm-config-management #
+```
+
+```
+ncn-m001:/home/csm-config-management #  kubectl logs -n services  cfs-b840455c-e919-4656-b64a-44d433f082dc-5d4jp ansible
+Waiting for Inventory
+Inventory generation completed
+SSH keys migrated to /root/.ssh
+...
+Sidecar available
+Running ansible/config_sbps_iscsi_targets.yml from repo https://api-gw-service-nmn.local/vcs/cray/csm-config-management.git
+....
+All playbooks completed successfully
+ncn-m001:/home/csm-config-management #
+```
+### **Important things to check after worker node(s) personalization**
+
+#### **Check to see if the SBPS Marshal Agent is running without any errors.**
+
+**E.g.**
+
+```
+ncn-w001:/home # systemctl status sbps-marshal
+● sbps-marshal.service - System service that manages Squashfs images projected via iSCSI for IMS, PE, and other ancillary images similar to PE.
+     Loaded: loaded (/usr/lib/systemd/system/sbps-marshal.service; enabled; vendor preset: disabled)
+     Active: active (running) since Thu 2024-08-22 11:57:48 UTC; 2 weeks 4 days ago
+   Main PID: 2878373 (sbps-marshal)
+      Tasks: 1
+     CGroup: /system.slice/sbps-marshal.service
+             └─ 2878373 /usr/lib/sbps-marshal/bin/python /usr/lib/sbps-marshal/bin/sbps-marshal
+...
+ncn-w001:/home #
+```
+
+#### **Check to see if the images/ LUN mapping created under "targetcli ls" cmd**
+Check to see if the “targetcli ls” output shows that fileio backing store are created for rootfs images along with corresponding iSCSI LUNs

```

Comment:
It's subtle to see, but the double quotes in `“targetcli ls”` are fancy multi-character sequences, please change them to simple double quotes like `"targetcli ls"` (like they are on the previous line)

Reply:
Fixed